ccd-definition/CaseField.json

```diff
@@ -136,5 +136,32 @@
     "Label": "Response deadline",
     "FieldType": "DateTime",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionProposedDeadline",
+    "Label": "Proposed new deadline",
+    "HintText": "This can't be later than 28 working days after the current deadline. All deadlines require a response before 4pm on the deadline day.",
+    "FieldType": "Date",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionAlreadyAgreed",
+    "Label": "Have you already agreed the new deadline with the claimant's legal representative?",
+    "HintText": "If not, you can propose it using this service.",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionReason",
+    "Label": "Reason why more time is needed.",
+    "HintText": "This is optional - the claimant will see your answer.",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
```

Comment:
@sabahirfan - Can we chat about the naming for these attributes? I think we need to qualify them a little more so we know specifically what the extension is for. There may be other things in this service or in other services where extensions are required and we should name them specifically. 

Also, can you confirm that these attributes are flattened in CaseField and not a ComplexType because you're splitting them across pages in the UI for capture?

Reply:
Yes, those are flattened in CaseField. I will catch up with you on naming for those offline.

Reply:
we could make these fields something like responseExtensionProposedDeadline etc maybe?

Reply:
I'm just checking with the business. I'm thinking whether we need to include info such as {party}{integer}{thingtobeextended}{attribute}{integer} - e.g. defendantSolictor2claimResponseExtensionProposedDeadline1

Reply:
@sabahirfan / @swalker125 - I've spoken with the business and this seems like the most appropriate naming at present - defendantSolictor1claimResponseExtensionProposedDeadline, defendantSolictor1claimResponseExtensionAlreadyAgreed, defendantSolictor1claimResponseExtensionReason

This is on the basis that there each defendant solicitor can submit separate extension requests, and each can only submit one.

When multi-party comes up, the integer after the party can change for the new set of attributes.

Reply:
renamed as suggested.